metriq_gym/process.py

```diff
@@ -8,7 +8,7 @@
 from qiskit_ibm_runtime import QiskitRuntimeService
 from qiskit.providers import Job, JobStatus
 
-from metriq_gym.bench import BenchJobResult, BenchProvider
+from metriq_gym.bench import BenchJobResult, BenchJobType, BenchProvider
 
```

Comment:
We should also 

```py
import logging
logging.basicConfig(level=logging.INFO)
```

as `logging` is used down below.

Reply:
For the one line of logging, I'd rather cut it.